distribution/docs/src/main/jdocs/content/_security-services/external-ws-s-sts-support-contents.adoc

```diff
@@ -0,0 +1,16 @@
+:type: securitySts
+:status: published
+:title: External/WS-S STS Support
+:link: _external_ws_s_sts_support
+:order: 02
+
+===== Security STS WSS
+
+This configuration works just like the STS Client Config for the internal STS, but produces standard requests instead of the custom ${branding} ones.
+It supports two new auth types for the context policy manager, WSSBASIC and WSSPKI.
```

Comment:
We might want to just add in a comment saying that one would use these auth types when connecting to a non-DDF STS. They can also be used with the DDF STS if you want to ignore realms.